eiffel-syntax-and-usage/the-links-object.md

```diff
@@ -75,7 +75,7 @@ __Description:__ Identifies the composition from which an artifact declared by [
 
 ### ENVIRONMENT
 __Required in:__ None  
-__Optional in:__ [EiffelArtifactCreatedEvent](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md), 
+__Optional in:__ [EiffelArtifactCreatedEvent](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md), [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md), 
```

Comment:
Sorry for late reply, I've been out of the office...

1. Yes, makes sense.
2. Wait, last time we talked about this we said that we should let TERC and TestSuite map 1:1. So if you're using TERC events, you should wrap all test case executions following that in a single TestSuite (hence the TestSuite has a TERC link type). Now you're suggesting that each recipe within the TERC corresponds to a separate TestSuite - there I think is the basic problem.
3. I think this pretty much just follows from the above problem.

I'll try to schedule a whiteboard session so we can hash this out.